app/pb_kits/playbook/pb_date/_date.jsx

```diff
@@ -1,21 +1,59 @@
+/* @flow */
+
 import React from 'react';
-import PropTypes from "prop-types";
-
-const propTypes = {
-  className: PropTypes.string,
-  id: PropTypes.string
-};
-
-class Date extends React.Component {
-  render() {
-    return (
-      <div className="pb_date">
-        <span>DATE CONTENT</span>
-      </div>
-    )
-  }
+import DateTime from '../pb_kit/dateTime.js'
+import moment from 'moment'
+require('moment-strftime')
+require('moment-timezone')
+
+import {
+  Body,
+  Icon,
+} from '../'
+
+type DateProps = {
+  className: String,
+  size?: 'xs' | 'md' | 'lg',
+  value?: String,
 }
 
-Date.propTypes = propTypes;
+const Date = (props: DateProps) => {
+  const {
+    size,
+    value,
+  } = props
+
+  const calendarIcon = function(size, value) {
+    const dateTimeValue = new DateTime({ value: value });
+
+    const monthAndDate = dateTimeValue.toMonth().toUpperCase().toString() + ' ' + dateTimeValue.toDay().toString()
+    const weekdayAndDot = moment(value).format('ddd').toUpperCase() + ' · '
```

Comment:
It may be worth adding these two lines as functions in `dateTime.js`. That would also eliminate the need for the `require` statements at the top.

Reply:
Do you think we'll want to use these strings somewhere else? Or are you suggesting the move just to make the code more organized?

Reply:
@drborges have you guys considered a transition to something like [date-fns](https://date-fns.org/) or some other suggestion from [here](https://github.com/you-dont-need/You-Dont-Need-Momentjs)? (I realize this might not be the place for that discussion)

Just curious 🤔